example/src/App.svelte

```diff
@@ -8,6 +8,16 @@
     <li><a href="/does/not/exist" use:link>Not found</a></li>
 </ul>
 
+<h2>Dynamic links</h2>
+<ul class="navigation-dynamic-links">
+{#each dynamicLinks as dl}
```

Comment:
Because of how Svelte works, the example doesn't work too well unless this line is replaced with:

````svelte
{#each dynamicLinks as dl (dl.id)}
````

Reply:
Done :)